geomstats/geometry/spd_matrices.py

```diff
@@ -22,12 +22,14 @@ def __init__(self, n):
             embedding_manifold=GeneralLinear(n=n))
         self.n = n
 
-    def belongs(self, mat, atol=TOLERANCE):
-        """Check if a matrix belongs to the manifold.
-
-        Check if a matrix belongs to the manifold of symmetric positive
-        definite matrices.
-        """
+    @staticmethod
+    def belongs(mat, atol=TOLERANCE):
+        """Check if a matrix is symmetric and invertible."""
+        # FIXME: check positivity, implying invertibility (Yann)
```

Comment:
If you want to attribute the FIXME to someone, use the syntax: 
```
# TODO(ythanwerdas): ...
```
Although, it's better practice to leave TODOs to yourself rather than to others.

TODOs and FIXMEs are automatically parsed by pycharm

Reply:
ok thanks :)

Reply:
does `# TODO(ythanwerdas, opeltre): ...` work ?